### PR TITLE
Pedalboard snapshot Indexes

### PR DESCRIPTION
**Pull request by falkTX**

### CODE REVIEW

File: html/js/banks.js [hunk 1]
```diff
@@ -192,6 +193,12 @@ JqueryClass('bankBox', {
             }, 500)
             self.data('savingTimeout', timeout)
         })
+        // Update displayed indexes
```
Comment (falkTX) [inline html/js/banks.js]: Why is this needed? It is being called inside the save function, which does not seem the right place for it.

File: html/js/banks.js [hunk 2]
```diff
@@ -345,10 +352,11 @@ JqueryClass('bankBox', {
         })
     },
 
-    renderPedalboard: function (pedalboard) {
+    renderPedalboard: function (pedalboard, index = "") {
         var self = $(this)
 
         var metadata = {
+            index: index ? index + ". " : index,
```
Comment (falkTX) [inline html/js/banks.js]: This line is quite confusing at first, I suggest to change to:
```
index: index ? (index + ". ") : "",
```

File: mod/host.py [hunk 3]
```diff
@@ -2360,8 +2360,10 @@ def snapshot_remove(self, idx):
         if idx < 0 or idx >= len(self.pedalboard_snapshots) or self.pedalboard_snapshots[idx] is None:
             return False
 
+        snapshot_to_remove = self.pedalboard_snapshots[idx]
         self.pedalboard_modified = True
-        self.pedalboard_snapshots[idx] = None
+        self.pedalboard_snapshots.remove(snapshot_to_remove)
+        logging.info(self.pedalboard_snapshots)
```
Comment (falkTX) [inline mod/host.py]: this logging is good for testing, but not in production.
change it to `logging.debug` please

File: html/js/banks.js (hunk 1 above)
Comment (blushi) [reply html/js/banks.js]: it's to cover all the cases at once: dragging, adding and removing banks, it would have to be called here https://github.com/moddevices/mod-ui/blob/indexes/html/js/banks.js#L78 (drag, add) and here https://github.com/moddevices/mod-ui/blob/indexes/html/js/banks.js#L90 (remove) instead
